docs/source/flow_setup.rst

```diff
@@ -222,6 +222,19 @@ required libraries as specified at
 <http://ray.readthedocs.io/en/latest/installation.html> and
 then follow the setup instructions.
 
+(Optional) Install Stable Baselines
+----------------------------
+
+An additional library that Flow supports is the fork of OpenAI's Baselines, Stable-Baselines.
+First visit <https://stable-baselines.readthedocs.io/en/master/guide/install.html> and
```

Comment:
techincally it's installed during the pip process. Maybe say something line. "This repository is already installed during the Flow installation procedure."